versions.json

```diff
@@ -3,7 +3,7 @@
         "commitish": "release-1.11.0",
         "version_string": "1.11.0",
         "license": "Apache License 2.0",
-        "consortium_build_number": "1",
+        "consortium_build_number": "0",
```

Comment:
Should this be a 2? See 0e87223b3f693bc26cdf5e7b146cce6f2a998529. Sorry if I missed this decision being made "offline"

Reply:
Good question.

@SwooshyCueb Thoughts? - When should the consortium_build_number be incremented vs reset?

Reply:
Bumped to 2.

Reply:
if avro is not getting its own version bumped... and depends on boost... then moving to 2 is correct.

if avro is itself getting a bump, then it should be restarted at 0.